### ISSUE TITLE
Add a Fake Panic Effect

### ISSUE THREAD

**Issue by mablue** [freqtrade/freqtrade#4626]
## Describe the enhancement
Add a Fake Panic Effect to back-test and hyper-opt 
When we set an order in reality in the market with lowest or highest price all other users can effect from that order and they wants to change the order.
This panic will make the order sell or buy harder than back-tests

*Explain the enhancement you would like*
We need an small panic effect on all orders in back-tests to find out exactly the order execute in true time or not.

**Comment by xmatthias** [comment]
It's unclear what you mean with "panic effect" - but we will not add a "falsifying" factor to backtesting results.

As explained in [backtesting assumptions](https://www.freqtrade.io/en/latest/backtesting/#assumptions-made-by-backtesting) - backtesting uses the data that's available (orders are opened at the candle's open value).

Backtesting assumes all orders are filled (depending on the volume on the exchange, this may not be the case in reality) at the desired price.

In short, you should not go from backtesting to live-trading directly, but battle-test your strategy using dry-run, as historic data may not represent the current / future market situation.

**Comment by mablue** [comment]
ok I undrestand we cant add new data to ohlcv.
But I see that allways when my bot set an order to the order list of real website other threaders panics and add smaller or bigger orders to change the order of orders list.
it never happening in dry run and my profit grow up massivly but in real im in loss

**Comment by xmatthias** [comment]
then maybe your order size is simply too big (or the markets are too small compared to your order sizes).

It'll also depend what type of order you use (market, limit).
A huge limit-order will serve as a buy/sell wall - so obviously other traders will react to it, while a market-order may itself move the price by eating up/down the orderbook.


Let's assume that your orders are "normally" sized - then it's not necessarily other traders that panic because of your order, it's just that the orderbook will constantly change (it is never static, on active markets it's not static for a millisecond ...). While this movement CAN be caused by your order - it usually isn't unless you're a whale trading huge sizes (what is huge will however depend on the exchange / market).
